src/ResourceManager/ContainerRegistry/Commands.ContainerRegistry/Commands/RemoveAzureContainerRegistry.cs

```diff
@@ -12,47 +12,51 @@
 // limitations under the License.
 // ----------------------------------------------------------------------------------
 
+using Microsoft.Azure.Commands.ResourceManager.Cmdlets.Components;
 using System.Management.Automation;
 
 namespace Microsoft.Azure.Commands.ContainerRegistry
 {
-    [Cmdlet(VerbsCommon.Remove, ContainerRegistryNoun,
-        DefaultParameterSetName = NameResourceGroupParameterSet,
-        SupportsShouldProcess = true)]
+    [Cmdlet(VerbsCommon.Remove, ContainerRegistryNoun, SupportsShouldProcess = true)]
```

Comment:
@Jinhuafei would you mind adding an `OutputType` to this cmdlet? The best practice for `Remove-*` cmdlets is to return either a `bool` (usually just true if the removal was successful) or the object that was removed.

This is usually paired with a `PassThru` switch parameter that you add. If the user sets the `PassThru` parameter, then you write to the output stream, and if they don't, then you return nothing.

Reply:
Fixed.